<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>train_view</class>
 <widget class="QWidget" name="train_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обучение сети</string>
  </property>
  <widget class="QPlainTextEdit" name="plain_text_edit_log">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>731</width>
     <height>111</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_train">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать обучение</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_box_train_params">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>341</width>
     <height>142</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры обучения</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_loss">
      <property name="text">
       <string>Функция ошибки:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="combo_box_losses">
      <item>
       <property name="text">
        <string>Бинарная кросс-энтропия</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Контрастная ошибка</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_optim">
      <property name="text">
       <string>Оптимизатор:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="combo_box_optims">
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Adam</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Adamax</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SGD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RMSprop</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_batch_size">
      <property name="text">
       <string>Размер мини-батча:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spin_box_batch_size">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_epochs_count">
      <property name="text">
       <string>Кол-во эпох:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spin_box_epochs_count">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="group_box_">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>731</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Результаты обучения</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_epochs_count_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Прогресс обучения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_test_model">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить обученную сеть</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
